frontend/ui/widget/inputdialog.lua

```diff
@@ -347,16 +348,20 @@ function InputDialog:init()
                                     - vspan_after_input_text:getSize().h
                                     - buttons_container:getSize().h
                                     - keyboard_height
-        if self.fullscreen or text_height > available_height then
+                                    - Size.padding.large -- keep some padding around the dialog
+        if self.fullscreen or self.use_available_height or text_height > available_height then
             -- Don't leave unusable space in the text widget, as the user could think
             -- it's an empty line: move that space in pads after and below (for centering)
+            local original_text_height = text_height
```

Comment:
Is new variable needed?

Reply:
Oh, I missed that `text_height` was local